cli/tests/unit/console_test.ts

```diff
@@ -831,7 +831,7 @@ unitTest(function consoleTestWithStringFormatSpecifier(): void {
 unitTest(function consoleTestWithObjectFormatSpecifier(): void {
   assertEquals(stringify("%o"), "%o");
   assertEquals(stringify("%o", 42), "42");
-  assertEquals(stringify("%o", "foo"), "foo");
+  assertEquals(stringify("%o", "foo"), `"foo"`);
```

Comment:
Do we want to keep %o and %s the same for strings?

Reply:
Node does quote and escape strings with the `%o` format specifier.

Reply:
Browsers do also quote strings with the %o specifier so this is indeed the correct behavior at-least in regards to quoting. Non printable characters differ across browsers.